packages/babel-plugin-polyfill-es-shims/data/polyfills.json

```diff
@@ -824,5 +823,174 @@
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.big": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.blink": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.bold": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.fixed": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.fontcolor": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.fontsize": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.italics": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.link": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.small": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.strike": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.sub": {
+    "chrome": "5",
+    "opera": "15",
+    "edge": "12",
+    "firefox": "17",
+    "safari": "6",
+    "node": "0.10",
+    "android": "4",
+    "ios": "7",
+    "phantom": "2",
+    "samsung": "1",
+    "electron": "0.20"
+  },
+  "String.prototype.sup": {
```

Comment:
I believe all of these work in ie 10 and 11 (but haven’t checked lately)

Reply:
I don't have access to IE to test it, but this data is generated from `compat-table`.

Reply:
Oops, I’ll update the compat table then, please hold :-)

Reply:
The data in the compat table under "name: 'String.prototype HTML methods'" includes `ie10: true,` for one test, and `ie11: false,` for another two which means all IE versions require the polyfill. What am i missing?

Reply:
It might be a bug in the script which extracts the data, let me check :thinking:

Reply:
Ahhh, this file contains _supported_ versions. IE is not marked as supported, so it always include the polyfill.

Reply:
I added a test

Reply:
ahhh ok thanks, that wasn't clear to me